ignition/getting-started-with-ignition.md

```diff
@@ -0,0 +1,95 @@
+# Getting Started with Ignition #
+
+Before getting started, make sure that Ignition is supported on your platform.
+For a full list of supported platforms, refer [here][supported platforms].
+
+Ignition uses a JSON config to represent the set of changes to be made. The
+format of this config is detailed [here][config format]. One of the most
+important parts of this config is the version number. This **must** match the
+version number accepted by Ignition. If the config version isn't accepted by
+Ignition, Ignition will fail to run and prevent the machine from booting. This
+can be seen by inspecting the console output of the failed instance. For more
+information, check out the [troubleshooting](#Troubleshooting) section.
+
+Let's take a look at a few example configs.
+
+[supported platforms]: supported-platforms.md
+[config format]: https://github.com/coreos/ignition/blob/master/doc/ignition.md#configuration
+
+## Example Configs ##
+
+Each of these examples is written in version 1 of the config. Always double
+check to make sure that your config matches the version that Ignition is
+expecting.
+
+### BTRFS Root ###
```

Comment:
Overall, can we label these by the topic they are showing not the specific example? For example, this one is called something like "Format and Mount Filesystems"